spec/book_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './book'
+
+describe 'Check if instance of class Book is true' do
+  context 'should create new class books' do
+    book = Book.new(title: 'Rising Sun', author: 'Dire Akanbi')
+
+    it 'should have class book' do
+      expect(book.class) == Book
+    end
+
+    it 'should have the correct title and author' do
+      expect(book.title).to eq 'Rising Sun'
+      expect(book.author).to eq 'Dire Akanbi'
+    end
+  end
```

Comment:
- [ ] Following the project requirement, you should add unit tests for all the methods in all the classes (as ex: add_rental, add_student, can_use_services?, of_age? ...).